Header header
float32 Zrobot
float32 Xrobot
uint8 label
float32[] covariance
